macros/attribution_overview.sql

```diff
@@ -64,9 +122,8 @@ with campaign_prep as (
   from {{ ref('snowplow_attribution_channel_attributions') }} c
   
   {% if var('snowplow__spend_source') != 'not defined' %}
-    left join {{ var('snowplow__spend_source') }} s
-    on c.channel = s.channel and s.spend_tstamp < cv_tstamp 
-    and s.spend_tstamp > {{ snowplow_utils.timestamp_add('day', -90, 'cv_tstamp') }}
```

Comment:
I do wonder if we could support a more "windowed" spend table, e.g. something like channel, campaign, spend, start_tstamp, end_tstamp and then do a join over that window so they can keep incrementing onto their spend table... bit more work and requires a better validation of the format (maybe we do upfront, like we do in unified?), but feels like it's a nicer experience for the user long term maybe?

Reply:
I always imagined the spend source as a view instead of an incremental table as most likely they have this information in some shape and form already and they would just have to align the column names to ours. As this would also change the app I feel like this is a good research topic to explore when collecting feedback from users and add it as v2 if it makes their lives easier. Let's discuss separately.